extensions/amp-list/amp-list.md

```diff
@@ -57,7 +57,7 @@ response and accessed using, e.g. `items="field1.field2"` expression.
 
 Thus, when `items="items"` is specified (the default) the response must be a JSON object that
 contains an array property "items":
-```json
+```text
```

Comment:
Pygments renders errors in red, which looks horrible. I felt in this case, nothing was lost by having it defined as text, and it got rid of the annoying red highlighting.  I have that as a discussion point to talk to you about.